### DISCUSSION TITLE
Create a communication channel.

### DISCUSSION THREAD

**Discussion by myusko** [Neoteroi/BlackSheep#136] [Ideas]
Hey, @RobertoPrevato probably would be cool to create a communication channel for BlackSheep, where the community and contributors can elaborate/discuss/ask, etc.

Possible good clients

- Discord
- Gitter

What do you think, makes sense?

**Comment by RobertoPrevato** [comment]
Hi @myusko, 
sorry for replying late. I like your idea, and yes it makes sense. I don't have experience in neither Discord or Gitter, but lately I wanted to try Discord; it will be interesting to check both of them.
Eventually, would you have time to help with administering the channel?

**Comment by myusko** [comment]
Yeah, sure, I can.

**Comment by myusko** [comment]
Once you create a channel or will need help/support, let me know.

**Comment by RobertoPrevato** [comment]
Thank you! I didn't have time yet to check this. I spent some time investigating an issue in blacksheep today, I am also still working on the [demo project](https://github.com/Neoteroi/Venezia/tree/dev).

Do you have any recommendation over Discord vs Gitter? I am leaning towards Discord, but I don't know them enough to have a clear preference.

**Reply by RobertoPrevato** [reply]
I just converted this into a discussion since it's not really an issue 😄

**Comment by myusko** [comment]
I love both clients, but probably right now I'd prefer Gitter, since the app, integrated with Github very well.

**Reply by RobertoPrevato** [reply]
@myusko I just created a Gitter community for Neoteroi and a channel for BlackSheep :)

[![Gitter](https://badges.gitter.im/Neoteroi/BlackSheep.svg)](https://gitter.im/Neoteroi/BlackSheep?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)

**Reply by myusko** [reply]
@RobertoPrevato I'm already there:)
